documentation/source/troubleshooting.rst

```diff
@@ -144,3 +144,16 @@ When you install cocotb, ``pip`` uses the system (or some other) compiler that s
 But when you try to run cocotb with the older Questa, it prepends the older libraries Questa ships with to ``LD_LIBRARY_PATH``.
 This causes the older libstdc++ Questa ships with to be loaded, resuling in the error message.
 For Questa, you can use the ``-noautoldlibpath`` option to turn off the ``LD_LIBRARY_PATH`` prepend to resolve this issue.
+
+
+Using cocotb with more than one Python installation
+===================================================
+
+Users of cocotb with more than one installation of a single Python version (including conda env users) must take care not to re-use cached versions of the installed cocotb package.
+If this step is omitted, running simulations fails with errors like ``libpython3.7m.so.1.0: cannot open shared object file: No such file or directory``.
```

Comment:
"this step" sounds like the user should execute a command, but there no such mention if it here.

Reply:
Slightly reworded in an update.